CHANGELOG.md

```diff
@@ -110,6 +110,7 @@ All notable, unreleased changes to this project will be documented in this file.
   - `channel` from `DraftOrderCreateInput` changed to channelId
   - `channel` from `DraftOrderInput` changed to channelId
   - `channel` from `pluginUpdate` changed to channelId
+- Compress celery tasks `send_fulfillment_confirmation_email_task`, `send_fulfillment_update_email_task`, `trigger_webhooks_for_event`, `send_webhook_request` - #7445 by d-wysocki
```

Comment:
Changelog is not up-to-date. IMHO no need to explicitly list all tasks which received the compression. You can just add info that user_emails, webhooks etc.